src/main/java/org/cascadebot/cascadebot/commands/subcommands/guild/GuildInfoSubCommand.java

```diff
@@ -0,0 +1,69 @@
+package org.cascadebot.cascadebot.commands.subcommands.guild;
+
+import net.dv8tion.jda.core.EmbedBuilder;
+import net.dv8tion.jda.core.entities.Guild;
+import net.dv8tion.jda.core.entities.Member;
+import org.cascadebot.cascadebot.CascadeBot;
+import org.cascadebot.cascadebot.UnicodeConstants;
+import org.cascadebot.cascadebot.commandmeta.CommandContext;
+import org.cascadebot.cascadebot.commandmeta.ICommandExecutable;
+import org.cascadebot.cascadebot.data.managers.GuildDataManager;
+import org.cascadebot.cascadebot.data.objects.Flag;
+import org.cascadebot.cascadebot.data.objects.GuildData;
+import org.cascadebot.cascadebot.messaging.MessagingObjects;
+import org.cascadebot.cascadebot.permissions.CascadePermission;
+import org.cascadebot.cascadebot.utils.FormatUtils;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+public class GuildInfoSubCommand implements ICommandExecutable {
+
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
+        GuildData dataForList = context.getData();
+        Guild guildForList = context.getGuild();
+
+        if (context.getArgs().length > 0) {
+            guildForList = CascadeBot.INS.getShardManager().getGuildById(context.getArg(0));
+            dataForList = GuildDataManager.getGuildData(Long.valueOf(context.getArg(0)));
```

Comment:
We need to do more error handling here such as:
 - What if the arg isn't a number
 - What if the guild can't be found (It should be used like this `GuildDataManager.getGuildData(guildForList.getIdLong());`)